didcomm/common/types.py

```diff
@@ -37,5 +37,108 @@ class VerificationMaterial:
 
 class DIDCommMessageTypes(Enum):
     ENCRYPTED = "application/didcomm-encrypted+json"
+    ENCRYPTED_SHORT = "didcomm-encrypted+json"
     SIGNED = "application/didcomm-signed+json"
+    SIGNED_SHORT = "didcomm-signed+json"
     PLAINTEXT = "application/didcomm-plain+json"
+    PLAINTEXT_SHORT = "didcomm-plain+json"
+
+
+DIDCommMessageMediaTypes = DIDCommMessageTypes
+
+# TODO
+#   - replace DIDCommMessageTypes with DIDCommMessageMediaTypes
+#   - rename DIDCommMessageProtocolTypes to DIDCommMessageTypes
+
+
+class DIDCommMessageProtocolTypes(Enum):
+    FORWARD = "https://didcomm.org/routing/2.0/forward",
+
+
+class MessageFields:
```

Comment:
1) Do we really need all the fields below? I think JOSE-related fields are checked bu JOSE lib, and we don't care about it, do we?
2) I would keep the Message-specific fields (JWM + DIDComm fields) in the Message class, not here.
3) Do I understand correctly that we separated this field names for the TBD validation of the Message during unpack/from_dict?

Reply:
the main intention here is to avoid many hard-coded strings in the code, so it's not just about in-validation usage but any referencing

Reply:
done but it might need additional discussion or later addressing